lib/tasks/bower-install.js

```diff
@@ -21,7 +22,7 @@ module.exports = Task.extend({
 
     ui.startProgress(chalk.green('Installing browser packages via Bower'), chalk.green('.'));
 
-    var config = bowerConfig.read();
+    var config = bowerConfig.read(path.resolve('.'));
```

Comment:
I'm wondering if we should use `this.project.root` here instead? Would that work for you too?